docs/advanced_topics/privacy.md

```diff
@@ -72,6 +72,28 @@ WAGTAIL_FRONTEND_LOGIN_URL = '/accounts/login/'
 
 To integrate Wagtail into a Django site with an existing login mechanism, setting `WAGTAIL_FRONTEND_LOGIN_URL = LOGIN_URL` will usually be sufficient.
 
+## Setting the default privacy restriction
+
+You can modify the default privacy restriction of a page by overriding the `get_default_privacy_setting` method in the Page model. This is often done to make a page require login by default, but it can also be used for more complex configurations, such as adjusting the default privacy setting based on the user or using an auto-generated shared password.
```

Comment:
Suggestion: It would be good to link to the reference entry for `get_default_privacy_setting` here, to cover off which values can be returned.

Reply:
Try this.

```suggestion
You can modify the default privacy restriction of a page by overriding the {meth}`~wagtail.models.Page.get_default_privacy_setting method in the Page model. This is often done to make a page require login by default, but it can also be used for more complex configurations, such as adjusting the default privacy setting based on the user or using an auto-generated shared password.
```

See https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#reference-roles